src/transaction/log_global.c

```diff
@@ -73,6 +73,7 @@ log_global::log_global ()
      }
   , group_commit_info LOG_GROUP_COMMIT_INFO_INITIALIZER
   , m_tran_complete_mgr (NULL)    // to be instantiated on boot
+  , m_ack_stream_position (std::numeric_limits<cubstream::stream_position>::max ())
```

Comment:
m_ack_stream_position should be 0

Reply:
it is temporary max to make recovery work with commit replicated sysops. should be properly initialized when m_ack_stream_position is management is mature.